crt_portal/cts_forms/templates/forms/report_base.html

```diff
@@ -112,5 +112,11 @@ <h2 class="margin-0" tabindex="0">{{ current_step_title }}</h2>
 {% endblock %}
 
 {% block page_js %}
+<script type="text/javascript">
+  var wordRemainingText = "{{ word_count_text.wordRemainingText }}"
```

Comment:
All of these variables should be put into an object, and probably wrapped in a closure to prevent them from leaking into the global scope. So:

```
(function(root) {
  root.CRT = root.CRT || {};
  root.CRT.translationStrings = {
   wordRemaining: ...,
   wordsRemaining: ...
  ...etc
  };
})(window);
```

Then, they can be referenced in `word_count` with `window.CRT.translationStrings.wordRemaining`, and so on.

Reply:
sounds good I can make that change

Reply:
I am not getting this working locally, If you have time tomorrow @el-mapache could you help me with it? thanks

Reply:
Definitely!